secio/src/error.rs

```diff
@@ -98,6 +98,7 @@ impl From<ring::error::Unspecified> for SecioError {
 }
 
 impl error::Error for SecioError {
+    #[allow(deprecated)]
```

Comment:
Any reason that we continue to use the deprecated interface?

Reply:
> Any reason that we continue to use the deprecated interface?

I understand wrong here, here can use `fn source` directly, no need to use the deprecated function